apps/web/src/components/score/ScoreVisual.tsx

```diff
@@ -2,89 +2,49 @@ import ScoreProfile from './ScoreProfile';
 import ScoreInfo from './ScoreInfo';
 import { UserScore } from 'types/Score';
 import { User } from 'types/User';
-import { Text, Image, Box, useMediaQuery, Flex } from '@chakra-ui/react';
+import { Text, Image, useMediaQuery, Box } from '@chakra-ui/react';
 import { useEffect, useState } from 'react';
-import { mobileData, defaultData } from './ScoreConfig';
 import ScoreColored from './ScoreColored';
 import ScoreColoredMobile from './ScoreColoredMobile';
+import { SSeraphiniStyled, TicketStyled } from './ScoreStyle';
 
 type Props = {
   size?: number;
   user?: User;
   userScore: UserScore;
 };
 
-export default function ScoreVisual({ size = 1, user, userScore }: Props) {
-  const [isLargerThan768] = useMediaQuery('(min-width: 768px)');
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const [cssProps, setCssProps] = useState<Record<string, any>>(defaultData);
+export default function ScoreVisual({ user, userScore }: Props) {
+  const [isLargerThan768] = useMediaQuery('(min-width: 769px)');
```

Comment:
I suggest using "isDesktop" or a more understandable name than "isLargerThan768".
What is larger? 768 pixels? Width or height?

Reply:
Agreed. However, since [`useMediaQuery` isn't intended for use in server-side rendered components](https://github.com/chakra-ui/chakra-ui/issues/3124#issuecomment-813809175), I think this change could be added to the backlog and just kept the current variable name 👀